backend/business-partner-agent/src/main/java/org/hyperledger/bpa/client/AcaPyLivenessCheck.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/business-partner-agent
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hyperledger.bpa.client;
+
+import java.util.Map;
+
+import javax.inject.Singleton;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.health.HealthStatus;
+import io.micronaut.management.endpoint.health.HealthEndpoint;
+import io.micronaut.management.health.indicator.AbstractHealthIndicator;
+import io.micronaut.management.health.indicator.annotation.Liveness;
+
+@Singleton
+@Liveness
+@Requires(beans = HealthEndpoint.class)
+public class AcaPyLivenessCheck extends AbstractHealthIndicator<Map<String, String>> {
+    @Override
+    protected Map<String, String> getHealthInformation() {
+        this.healthStatus = HealthStatus.UP;
```

Comment:
implementation?

Reply:
if this can we served it means it is live. should I check anything else?
Acapy liveness I explicitly not check

Reply:
Then I would rename it to BPALivenessCheck, because otherwise its confusing.